test/v1alpha1/timeout_test.go

```diff
@@ -174,6 +174,7 @@ spec:
 // TestTaskRunTimeout is an integration test that will verify a TaskRun can be timed out.
 func TestTaskRunTimeout(t *testing.T) {
 	t.Parallel()
+	timeout := 1 * time.Second
```

Comment:
seems a bit odd to me to create a timeout var and in the next line immediately use a different timeout?

Reply:
Good point, I've named the two different timeouts with vars to make it a bit clearer.